ambassador/ambassador/ir/irhttpmappinggroup.py

```diff
@@ -332,7 +332,7 @@ def finalize(self, ir: 'IR', aconf: Config) -> List[IRCluster]:
                 for mapping in self.mappings:
                     if 'weight' not in mapping:
                         mapping.weight = round((100.0 - total_weight)/unspecified_mappings)
-            elif total_weight != 100.0:
+            elif 0 < total_weight < 100.0:
```

Comment:
What happens if `total_weight > 100.0`?

Reply:
@ark3 that's a fine question, good catch :) here comes an update